src/general/smoothing_kernels.jl

```diff
@@ -473,24 +426,13 @@ on the dimension. The normalization factor `` \sigma `` is `` 9 / \pi `` in two
 
 This kernel function has a compact support of `` [0, h] ``.
 
-For a detailed discussion on Wendland functions and their applications in SPH, see (Dehnen & Aly, 2012).
+For a detailed discussion on Wendland functions and their applications in SPH, see [Dehnen2012](@cite).
 The smoothness of these functions is also the largest disadvantage as they loose details at sharp corners.
 
 The smoothing length is typically in the range ``[3.0\delta, 4.5\delta]``,
 where ``\delta`` is the typical particle spacing.
 
 For general information and usage see [Smoothing Kernels](@ref smoothing_kernel).
-
-## References
-- Walter Dehnen & Hassan Aly.
-  "Improving convergence in smoothed particle hydrodynamics simulations without pairing instability".
-  In: Monthly Notices of the Royal Astronomical Society 425.2 (2012), pages 1068-1082.
-  [doi: 10.1111/j.1365-2966.2012.21439.x](https://doi.org/10.1111/j.1365-2966.2012.21439.x)
-
-- Holger Wendland.
```

Comment:
Wendland has to mentioned somewhere for the Wendland kernel. Either in text, or keep the references section.